terraform/implementation/ecs/backend.tf

```diff
@@ -0,0 +1,28 @@
+terraform {
+  required_providers {
+    aws = {
+      source  = "hashicorp/aws"
+      version = "=5.31.0"
+    }
+  }
+
+  backend "s3" {
+    key     = "remote_tfstate"
+    encrypt = true
+    # dynamodb_table
```

Comment:
Do we need this commented out code for the backend `s3` bucket?

Reply:
I'd like to leave these here for reference. These values are variable depending on how things in the `/setup` directory go and we can't set variables in this block. We have to pass the values like we do on line 111 in the `ecs.sh` script.